### ISSUE TITLE
PWA overwritting settings

### ISSUE THREAD

**Issue by mouadessalim** [flet-dev/flet#1499]
**Description**

I'm trying to modify the manifest file to match my project, but the problem is that everything works except the background color. After some research I noticed that the color is updated during the launch of the PWA app but is directly replaced by the default blue of Flet. In my opinion, this is due to the default manifest integrated by flet. The project name is updated but the color reverts to the default blue for no reason. 

**Code example to reproduce the issue:**

```json
{
  "name": "SimpleCoinMarket",
  "short_name": "SimpleCoinMarket",
  "start_url": ".",
  "display": "standalone",
  "background_color": "#FF0000",
  "theme_color": "#FF0000",
  "description": "SimpleCoinMarket description",
  "orientation": "natural",
  "prefer_related_applications": false,
  "icons": [
    {
      "src": "icons/icon-192.png",
      "sizes": "192x192",
      "type": "image/png"
    },
    {
      "src": "icons/icon-512.png",
      "sizes": "512x512",
      "type": "image/png"
    },
    {
      "src": "icons/icon-maskable-192.png",
      "sizes": "192x192",
      "type": "image/png",
      "purpose": "maskable"
    },
    {
      "src": "icons/icon-maskable-512.png",
      "sizes": "512x512",
      "type": "image/png",
      "purpose": "maskable"
    }
  ]
}
```

**Describe the results you received:**

https://github.com/flet-dev/flet/assets/38190847/cf74bec1-80a9-4df8-b4d6-882f0f8b7e81

> You can see me during the multiple attempts than during the seconds : `0:28`, `0:31`, `0:34` that the background color is actually red (#FF0000). 

**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.7.4
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page: 
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages
Requires: flet-core, httpx, oauthlib, packaging, watchdog, websocket-client, websockets
```

**Operating system**:

```
ProductName:		macOS
ProductVersion:		13.2
BuildVersion:		22D49
```
> I've already tried it with several browsers as well as several devices on different OS: On Windows 10 and IOS

**Additional information you deem important (e.g. issue happens only occasionally):**

If you have the slightest doubt that the problem comes from my project, I invite you to create the simplest application with a different background color instead of the default flet blue.

**Comment by FeodorFitsner** [comment]
There is certainly no magic in there. I'm trying here and indeed it takes app name and description from `manifest.json`. And when you open `{app_url}/manifest.json` in the browser you can clearly see it's pulling the correct one with right colors. Must be something else...

**Comment by mouadessalim** [comment]
mhh, i see thanks for you help ! Do you have any idea on how to resolve the problem, i've published the project on flet servers so if you want to test you can try here : https://simplecoinmarket.fly.dev/

**Comment by FeodorFitsner** [comment]
Yeah, I knew it! The following must be added to `<head>` in `index.html`:

```html
<meta name="theme-color" content="#ff0000">
```

I got it from [here](https://stackoverflow.com/a/74847072/1435891). Don't ask me why - it's the same reason iOS grabs app icon from `apple-touch-icon` meta only.

**Comment by mouadessalim** [comment]
ok i see know, it make sens, so basilcy there is a way to resolve the problem using static website but if you use it with a server side how can i acess the html file to make all changes that i want

**Comment by FeodorFitsner** [comment]
The same approach as with `manifest.json` - grab original `index.html` from [here](https://github.com/flet-dev/flet/blob/main/client/web/index.html), modify it and put into your `assets` dir.

**Comment by mouadessalim** [comment]
@FeodorFitsner If we replace the index.html, it's absolutly broking everything, i even tried to import the whole web folder but it broke everything, not able to find the flutter dart files 
<img width="1440" alt="Capture d’écran 2023-06-18 à 23 17 05" src="https://github.com/flet-dev/flet/assets/38190847/e6a29e2c-fe6c-4325-a725-5cff760dc160">

**Comment by FeodorFitsner** [comment]
You have to take exactly that `index.html` from the repo, not just "View source and then copy paste", as it has some template strings that are replaced on the server side.

**Comment by mouadessalim** [comment]
exactly 
<img width="1373" alt="Capture d’écran 2023-06-18 à 23 25 09" src="https://github.com/flet-dev/flet/assets/38190847/0c897b2e-0f8a-4df9-b8ba-233939b1a4b4">

**Comment by mouadessalim** [comment]
My problem is solved and more, I can now change the title and description from the source. I'm not going to close the issue so you can integrate it dynamically by referencing it in a commit.

**Comment by mouadessalim** [comment]
Dear @FeodorFitsner ,

I wanted to take a moment to express my heartfelt gratitude and sincere thanks for the incredible work you have done with your project on GitHub. Your contribution is simply outstanding! With over 180,000 lines of commits, your dedication and coding prowess are truly commendable.

I want you to know that your project has had an incredible impact on many people, myself included. The features you have developed and the quality of your code are truly remarkable. Your work has solved real-world problems and made the lives of numerous users easier. It's an impressive achievement, and I wanted to congratulate you on that.

However, I would also like to encourage you to consider seeking help. I understand that undertaking such a massive project alone can be extremely challenging. While you have already accomplished so much, sharing the workload with other talented developers could lighten your burden and allow the project to progress even faster. Your expertise combined with that of other contributors could give rise to new innovative ideas and even more exciting developments.

In any case, I want to sincerely thank you for your dedication, expertise, and invaluable contribution to the development community. You are an exceptional developer, and your work has a real impact on many people. I wish you the very best for your future projects, and I can't wait to see the wonders you will continue to create.

Once again, thank you from the bottom of my heart!

Best regards,
Me

**Comment by ndonkoHenri** [comment]
Close issue?
